### PR TITLE
Finish to do list list structure

### PR DESCRIPTION
**Pull request by mustabbas**
### Change Done
Added a new project with webpack.
Created an index.html file.
Created an index.js file and set an array of some simple to-do tasks (array of objects).
Created a function to iterate over the tasks array and populate an HTML list item element for each task
Created a style.css and set rules for the To-Do List CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
All your source files (index.html, index.js and style.css) are located in /src directory and distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,71 @@
...
-### Install
+To get a local copy up and running follow these simple example steps.
 
-### Usage
+### Prerequisites
+Any modern web browser will do.
 
-### Run tests
+### Setup
+Follow the steps described before.
```
Comment (Lameck1) [inline README.md:39]: - [ ] These are now not necessary if they are pointing us to the details above.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,71 @@
...
+**If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
 
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
 
-To get a local copy up and running follow these simple example steps.
+git clone https://github.com/mustabbas/AwesomeBooks
+Once the files are on your machine, open the course folder in Visual Studio Code.
 
-### Prerequisites
+With the files open in Visual Studio Code, press the Go Live button at the bottom of the window to launch the files with Live Server.**
```
Comment (Lameck1) [inline README.md]: - [ ] Getting started needs improvement, this is not how we get started with a webpack project.

This is the procedure
1. Clone the repo(**tell us how**)
2. Navigate to project's directory(**tell us how**)
3. Install dependencies(**tell us how**)
4. Lastly start the project(**tell us how**)
    Two options here
     - `npm start` on the terminal OR
     - `npm run build` then navigate to build directory and open the index.html file in the browser

**Review by Lameck1 — COMMENTED** [review]
Hi @mustabbas, 

You've put in great effort in building your project. However, this code review request will have to be invalidated. We are the developers of tomorrow, and we are encouraged to practice and abide by industry standards and professionalism.

## Invalid Code Review Request

You have submitted a project, that does not have a professionally documented README. Suggestions are left in the review thread to help with making improvements to the README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 3]
```diff
@@ -1,75 +1,71 @@
...
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- I would like to thank the designer who design this website [Cindy Shin](https://www.behance.net/adagio07) [design guidelines,](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```
Comment (Lameck1) [inline README.md]: - [ ] This acknowledgment is invalid. Cindy Shin has not participated in this project in any way.

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,53 @@
...
+  margin: 0;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.addRow i {
+  position: absolute;
+  right: 30px !important;
+
+  /* padding: 5px; */
```
Comment (Dipeshtwis) [inline src/style.css]: - [ ] I would recommend removing the commented code to look codebase cleaner.

File: src/index.js [hunk 5]
```diff
@@ -1,13 +1,35 @@
...
-//     return element;
-//   }
-  
-//   document.body.appendChild(component());
\ No newline at end of file
+const listGroup = document.querySelector('.list-group');
+const ListArray = [
+  { id: 1, description: '1111111111111', isCompleted: false },
+  { id: 2, description: '2222222222222', isCompleted: false },
+  { id: 3, description: '3333333333333', isCompleted: false },
```
Comment (Dipeshtwis) [inline src/index.js]: [OPTIONAL]

- Always give meaningful task description to keep the app meaningful.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mustabbas,

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: configured project with webpack.
👍 The design is looking similar with the given design.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

Check the comments under the review


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Check the comments under the review

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @mustabbas, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**